src/pages/myPage.js

```diff
@@ -1,7 +1,217 @@
-import React from 'react';
-
-const MyPage = () => {
-  return <div>MyPage</div>;
-};
-
-export default MyPage;
+import React, { useCallback, useEffect, useState, useRef } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useNavigate } from 'react-router';
+import { removeCookie } from '../components/Shared/Cookies';
+import { getUserInfo } from '../store/myPage';
+import SideNavbar from 'components/SideNavbar';
+import LoginButton from 'components/Shared/LoginButton';
+import EditNickName from 'components/Mypage/EditNickName';
+import SchoolAuth from 'components/Mypage/SchoolAuth';
+import AutoLogin from 'components/Mypage/AutoLogin';
+import styled from 'styled-components';
+
+const MyPage = () => {
+  const toggle = useSelector((state) => state.toggle.isOpen); //사이드 네비게이션 상태 전역 상태 관리
+  const userInformation = useSelector((state) => state.myPage); //2.저장된 유저 데이터 불러온다.
+  const loginInfo = useSelector((state) => state.auth);
+  const [userImgFile, setUserImgFile] = useState(''); //기본 이미지 url 유저 정보안에 들어있어야함
+
+  const setFile = (e) => {
+    if (e.target.files[0]) {
+      const img = new FormData();
+      img.append('file', e.target.files[0]);
+      axios
+        .post('', img)
+        .then((res) => {
+          setImageUrl(res.data);
+        })
+        .catch((err) => {
+          console.error(err);
+        });
+    }
+  };
+  const photoInput = useRef();
+  const [isShown, setIsShown] = useState(false);
+  const dispatch = useDispatch();
+  const naviagate = useNavigate();
+  /*
+    TODO:
+    [] 유저정보 받아올 것: 이미지, 닉네임, 학교인증여부  
+  */

```

Comment:
주석이 너무 별로예요...
대관님만 알아볼 주석이 아니라 다같이 알아볼 주석을 만들어주세요

```suggestion
  /*
    TODO:
    - 이미지, 닉네임, 학교 인증 여부를 가져오는 로직을 추가해야 합니다.
  */

```